### PR TITLE
NVM & Node Setup Doc

### PR DESCRIPTION
**Pull request by SLAU925**
[SCSE-16]

### CODE REVIEW

File: docs/frontend/env-setup.mdx [hunk 1]
```diff
@@ -13,6 +13,8 @@ Do not install node.js outside of nvm. if you have node.js installed,
 you should uninstall it before proceeding with the nvm install
 :::
 
+Testing only
+
```
Comment (chanbakjsd) [inline docs/frontend/env-setup.mdx:17]: These seem to be unrelated to the setup doc.

File: docs/getting-started/node-nvm.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 2
+---
+
+# NVM and Node.Js
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:5]:
```suggestion
# NVM and Node.js
```
`Node.js` should only have N capitalized when it's capitalized.

File: docs/getting-started/node-nvm.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Uninstall Node.Js(if you have it installed outside of nvm)
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:7]:
```suggestion
## Uninstall Node.js (if you have it installed outside of nvm)
```

File: docs/getting-started/node-nvm.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:9]: Would it be a better idea to uninstall it using control panel or similar tools to be sure that the registry entries get removed as well?

File: docs/getting-started/node-nvm.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Installing the NVM 
+
+Head over to the [NVM Setup](https://www.freecodecamp.org/news/nvm-for-windows-how-to-download-and-install-node-version-manager-in-windows-10/) and follow all the guidelines and intructions
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:14]: Would be nice if the actual steps to install NVM are listed in this document as it seems directly related with the document itself.

File: docs/getting-started/node-nvm.mdx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+To install the latest version of Node, run: 
+```
+nvm install latest
+```
+
+You can also install different versions of Node:
+```
+nvm install node-version-number
+```
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:31]: Consider providing examples of how to specify the node version number (eg. `14`, `14.2` as opposed to `v14`).

**Review by chanbakjsd — CHANGES_REQUESTED** [review]
The setup doc is a good start but still requires a few minor changes.
Also consider separating installing and usage instructions (such as by grouping them into two headers) so that members can easily access the part they need.

File: docs/getting-started/node-nvm.mdx (hunk 4 above)
Comment (SLAU925) [reply docs/getting-started/node-nvm.mdx:9]: Should I just ask them to uninstall the node.js using control panel?

File: docs/getting-started/node-nvm.mdx (hunk 6 above)
Comment (SLAU925) [reply docs/getting-started/node-nvm.mdx]: Updated! kindly help me to check again :)

File: docs/getting-started/node-nvm.mdx (hunk 4 above)
Comment (chanbakjsd) [reply docs/getting-started/node-nvm.mdx:9]: That would be ideal unless there's a better way.

File: docs/getting-started/node-nvm.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+# NVM and Node.js
+
+## Uninstall Node.js(if you have it installed outside of nvm)
+
+Try to look for your current node.js file in `C:\Program Files` and uninstall it.
+
+
+## Installing the NVM 
+
+- Step 1: Head over to the [nvm-windows repository](https://github.com/coreybutler/nvm-windows#installation--upgrades) and click on **Download Now**
```
Comment (chanbakjsd) [inline docs/getting-started/node-nvm.mdx:14]: There are no "Download Now" buttons at the location linked. In addition, it'd be a good idea to use ordered list (`1.`/`2.`) instead of unordered list (`-`).

**Review by chanbakjsd — CHANGES_REQUESTED** [review]
Nice changes but the installation guide doesn't seem to reflect the current state of the repository.

**Comment by RealDyllon** [comment]
@jamiegoh can add win/mac tabs here as required, and add mac steps?

**Comment by jamiegoh** [comment]
PR superseded by #18
